src/components/Community/AddPost/PostInfo.tsx

```diff
@@ -0,0 +1,115 @@
+// import { db, collection, addDoc } from './firebaseConfig';
+import { useState } from 'react';
+import PostInfoSection from './PostInfoSection';
```

Comment:
아래와 같이 import 하는 모듈의 종류끼리 개행을 넣어주는 것도 좋습니다!
```suggestion
import { useState } from 'react';

import PostInfoSection from './PostInfoSection';
```